### PR TITLE
detect unknown fields in Kube-manifest.yaml

### PR DESCRIPTION
**Pull request by mengqiy**
This PR depends https://github.com/kinflate/tuthello/pull/2.

### CODE REVIEW

**Comment by mengqiy** [comment]
The tutorial test is failing until kinflate/tuthello#2 merged.

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: go
 
 go:
-  - 1.9.x
+  - 1.10.x
```
Comment (Liujingfang1) [inline .travis.yml]: Why do you need to bump go version?

Comment (mengqiy) [reply .travis.yml]: `DisallowUnknownFields` is introduced in golang 1.10.

Search `encoding/json` in the golang 1.10 [release notes](https://golang.org/doc/go1.10#minor_library_changes), you will see.

File: cmd/kinflate/README.md [hunk 2]
```diff
@@ -13,6 +13,8 @@ go get k8s.io/kubectl/cmd/kinflate
 
 After running the above command, `kinflate` should get installed in your `GOPATH/bin` directory.
 
+Note: `kinflate` requires 1.10 or newer
```
Comment (Liujingfang1) [inline cmd/kinflate/README.md]: Assume 1.10 is available. When running `kinflate inflate -f . | kubectl apply -f -`, kubectl is also using 1.10. Is that right?

Comment (mengqiy) [reply cmd/kinflate/README.md]: kubectl doesn't have to use golang 1.10. It's just a binary, which can be downloaded or compiled. 
Using golang 1.10 to compile should work fine, it should be backward compatible.

**Comment by mengqiy** [comment]
travis ci didn't report test status...

**Pull request merged**